cart.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles/csrt.css">
+</head>
+<body>
+  <nav>
+    <div class="cartPageNavLogo">
+        <img src="assets/84426.svg">
```

Comment:
good practice is to use the `alt` attribute with `img` tags to provide a fallback message if the image fails to load.

check this out: [alt attribute](https://www.geeksforgeeks.org/html-img-alt-attribute/)